listingslab-software/agile-pwa
# Feature/spinup new app

## QA
```
git checkout feature/spinup-new-app
rm -rf node_modules
yarn && yarn start
```
If you see anything at all, it passes
